http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -339,33 +343,47 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
     ev: Ask[F, AppContext]
   ): F[(ContainerName, WsmControlledResourceId)] = {
```

Comment:
nit: I prefer case classes  (i.e. `CreateStorageContainerResult`) over tuples wherever possible since it achieves the following, but not strictly necessary
- Makes the return type to functions stay the same if you want to change what is returned
- Makes accessing the values more clear as opposed to tuple deconstruction syntax